docs/KNOWNISSUES.md

```diff
@@ -2,138 +2,120 @@
-data is passed to the next command.
-(This behavior is intended to be fixed in a later release.)
+Use `Get-Content` to write the contents of a file into the pipeline.
 
 Redirected output will contain the Unicode byte order mark (BOM) when the default UTF-8 encoding is used.
 The BOM will cause problems when working with utilities that do not expect it or when appending to a file.
-
-Use `-Encoding ascii` to write ASCII text (which, not being Unicode, will not have a BOM).
+Use `-Encoding Ascii` to write ASCII text (which, not being Unicode, will not have a BOM).
+(Note: see [RFC0020](https://github.com/PowerShell/PowerShell-RFC/issues/71) to give us feedback on improving the encoding experience for PowerShell Core across all platforms.
+We are working to support UTF-8 without a BOM and potentially changing the encoding defaults for various cmdlets across platforms.)
 
 ### Job Control
 
 There is no job-control support in PowerShell on Linux/macOS.
 The `fg` and `bg` commands are not available.
-`Ctrl-Z` sends the `powershell` process to the background.
+
+For the time being, you can use [PowerShell jobs](https://msdn.microsoft.com/en-us/powershell/reference/5.1/microsoft.powershell.core/about/about_jobs) which do work across all platforms.
 
 ### Remoting Support
 
-Client-side remoting from Linux/macOS is not supported with the initial package.
-The work is being done in the [psl-omi-provider](https://github.com/PowerShell/psl-omi-provider) repo.
+Currently, PowerShell Core supports PowerShell Remoting (PSRP) over WSMan with Basic authentication on macOS and Linux, and with NTLM-based authentication on Linux.
+(Kerberos-based authentication is not supported.)
+
+The work for WSMan-based remoting is being done in the [psl-omi-provider](https://github.com/PowerShell/psl-omi-provider) repo.
+
+PowerShell Core also supports PowerShell Remoting (PSRP) over SSH on all platforms (Windows, macOS, and Linux).
+While this is not currently supported in production, you can learn more about setting this up [here](https://github.com/PowerShell/PowerShell/tree/master/demos/SSHRemoting).
 
 ### Just-Enough-Administration (JEA) Support
 
-The ability to create constrained administration (JEA) remoting
-endpoints is not currently available in PowerShell on Linux/macOS.
-This feature is currently not in scope for 6.0 and something we will consider post 6.0 but requires significant design work.
+The ability to create constrained administration (JEA) remoting endpoints is not currently available in PowerShell on Linux/macOS.
+This feature is currently not in scope for 6.0 and something we will consider post-6.0 as it requires significant design work.
 
 ### `sudo`, `exec`, and PowerShell
 
-Because PowerShell runs most commands in memory (like Python or Ruby)
-you can't use sudo directly with PowerShell built-ins.
+Because PowerShell runs most commands in memory (like Python or Ruby) you can't use sudo directly with PowerShell built-ins.
```

Comment:
missing comma after the introduction `Because PowerShell runs most commands in memory (like Python or Ruby)`